tests/accessor/accessor_common.h

```diff
@@ -843,7 +843,7 @@ void test_accessor_types_common() {
 }
 
 template <typename T, typename AccT, int dims>
-decltype(auto) get_subscript_overload(AccT& accessor, size_t index) {
+decltype(auto) get_subscript_overload(const AccT& accessor, size_t index) {
```

Comment:
I'm curious why this was a problem?

Reply:
@ProGTX It's a problem if accessor is const, since AccT is explicitly provided.